### PR TITLE
Network november release

### PR DESCRIPTION
**Pull request by seyedfayaz**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.

### CODE REVIEW

File: DdosCustomPolicyCreate.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
+{
+  "parameters": {
+    "api-version": "2018-11-01",
```
Comment (tejaskarandikar) [inline DdosCustomPolicyCreate.json]: Should probably be 2018-10-01?

Comment (tejaskarandikar) [reply DdosCustomPolicyCreate.json]: That is the API version for DDOS custom policy in NRP today

Comment (seyedfayaz) [reply DdosCustomPolicyCreate.json]: done.

**Comment by salameer** [comment]
We cannot accept this PR like that. You’ll need to add them under the Networking folders. Please have a sync with someone from the NRP Swagger Admins

Closing this, please open a new PR when ready

**Pull request closed without merging**